### DISCUSSION TITLE
Unchecked runtime.lastError

### DISCUSSION THREAD

**Discussion by cgofun** [WongSaang/chatgpt-ui#133] [General]
如果在对话框输入时粘贴剪切板，控制台会出现下面的错误，正常输入不会出现。
Unchecked runtime.lastError: The message port closed before a response was rece。
![image](https://user-images.githubusercontent.com/7732886/230750723-3bebf1e9-5293-4bbe-9317-5fd9da2477bf.png)

**Comment by WongSaang** [comment]
抱歉，我无法复现这个问题，还可以提供更多信息吗？
